packages/__docs__/package.json

```diff
@@ -86,6 +86,21 @@
     "@instructure/ui-tree-browser": "^7.7.0",
     "@instructure/ui-utils": "^7.7.0",
     "@instructure/ui-view": "^7.7.0",
+    "@instructure/ui-byline": "^7.7.0",
+    "@instructure/ui-truncate-text": "^7.7.0",
+    "@instructure/ui-metric": "^7.7.0",
+    "@instructure/ui-modal": "^7.7.0",
+    "@instructure/ui-radio-input": "^7.7.0",
+    "@instructure/ui-grid": "^7.7.0",
+    "@instructure/ui-checkbox": "^7.7.0",
+    "@instructure/ui-list": "^7.7.0",
+    "@instructure/ui-rating": "^7.7.0",
+    "@instructure/ui-text-area": "^7.7.0",
+    "@instructure/ui-flex": "^7.7.0",
+    "@instructure/ui-file-drop": "^7.7.0",
+    "@instructure/ui-link": "^7.7.0",
+    "@instructure/ui-heading": "^7.7.0",
+    "@instructure/ui-progress": "^7.7.0",
```

Comment:
Had to add these depenecies to the docs-app, because the build failed without them